tools/WinObjC.Tools/vsimporter-templates/project.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dependencies": {
+    "WinObjC.Frameworks": "*",
```

Comment:
It's just one more thing to go wrong, imo. I'm not totally against the idea though.

---
In reply to: [100940824](https://github.com/Microsoft/WinObjC/pull/1958#discussion_r100940824) [](ancestors = 100940824,100940735)